### ISSUE TITLE
在VOC2012.py中gt_classes为什么是1而不是select_class？

### ISSUE THREAD

**Issue by qwe3535** [showlab/DatasetDM#33]
你好，有个问题想请教一下，在VOC2012.py中gt_classes为什么是1而不是select_class？

instances = {}
mapper_classes = [1]
instances["gt_classes"] = torch.tensor(mapper_classes, dtype=torch.int64)
masks = []
masks.append(mask == 1)

https://github.com/showlab/DatasetDM/blob/7545a68f35d8aad37f7963c326ae7c759a9a89d2/dataset/VOC2012.py#L233C2-L234C82 #
